### PR TITLE
Add post about R-Consortium support for R-Universe

### PR DESCRIPTION
**Pull request by noamross**
Spanish translation still to-do

### CODE REVIEW

**Comment by noamross** [comment]
Spanish translation added, date changed, and a tweak based on `roblog` checks.  R Consortium will post on Dec 3, it owuld be good to put this up on the 3rd or 4th.

**Comment by yabellini** [comment]
I can review the Spanish after the English version is done (including the reviews).

**Comment by steffilazerte** [comment]
I can review today or tomorrow

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+have generously supported R-Universe for the past year. Now, in
+granting R-Universe top-level status, the R Consortium has committed to support
+us for three more years. This endorsement and investment will help us meet our goal 
+of making R-Universe a sustainable and scalable platform for R package
+development that individuals and organizations can rely on. Jeroen Ooms,
+R-Universe's lead developer, will also join the R-Consortium's Infrastructure
+Steering Committee. We are glad to play a part in helping support important
+R community projects in the future!
+
+R-universe is rOpenSci’s platform for improving publication and discovery of research software in R.
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:35]:
```suggestion
R-Universe is rOpenSci’s platform for improving publication and discovery of research software in R.
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+It serves as a **global catalog** of software, articles, and data found
+across R repositories.  It is **software publication platform** providing a fully
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:37]:
```suggestion
across R repositories.  It is a **software publication platform** providing a fully
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+automated pipeline for testing, building, and publishing R packages. It is also
+a **meta-repository**, providing common infrastructure for both individuals or
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:39]:
```suggestion
a **meta-repository**, providing common infrastructure for both individuals and
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+organizations to manage custom R package repositories using their own approach
+to curation, release management, and quality control.
+
+R-Universe already serves tens of thousands of packages published by over a thousand
+organizations, and we are excited for what's ahead. This year, our we aim to to improve features to serve groups that
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:44]:
```suggestion
organizations, and we are excited for what's ahead. This year, our aim is to improve features which serve groups that
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+are building large-scale, collaborative repositories using R-Universe's platform
+and API, like the new [R-Multiverse](https://r-multiverse.org/). 
+We also plan expand our contributor base through more developer documentation and
+partner outreach, and to to improve performance and reliability as we scale. You
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:48]:
```suggestion
partner outreach, and to improve performance and reliability as we scale. You
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+can read more about our plans in our R-Consortium 
+[grant proposal](https://github.com/r-universe-org/consortium).
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:50]: Wrong link?

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+slug: r-universe-r-consortium-tlp
+title: R-Universe Named R Consortium Top-Level Project
+author:
+  - Noam Ross
+  - Jeroen Ooms
+featured: true
```
Comment (steffilazerte) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:7]:
```suggestion
editor: Steffi LaZerte
featured: true
```

**Review by steffilazerte — COMMENTED** [review]
Hi @noamross, Hi @jeroen,

Looks good! Just a couple of missing/broken links and stutters. 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok  -> Assuming the Spanish is translated in time, otherwise remember to update both the yaml and the folder name
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'editor' is filled with your name
* [X] YAML field 'preface' is present if necessary

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md (hunk 6 above)
Comment (jeroen) [reply content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:50]: This is currently private. Not sure if we want to make this public?

**Comment by jeroen** [comment]
Added r-hub to the list of top level projects.

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+tags:
+  - r-universe
+  - grant
+  - funding
+  - infrastructure
+description: "We're excited to announce R-Universe has been named the R-Consortium's newest Top-Level Project."
+---
+
+We are excited to announce that [R-Universe](/r-universe/)
+has been named the [R Consortium's] newest [Top-Level Project](https://r-consortium.org/all-projects/)!  We're glad
```
[the remark this answers is not in the document]
Comment (noamross) [reply content/blog/2024-12-03-r-universe-r-consortium-tlp/index.md:18]:
```suggestion
has been named the [R Consortium's](https://r-consortium.org) newest [Top-Level Project](https://r-consortium.org/all-projects/)!  We're glad
```

**Comment by noamross** [comment]
Thanks @steffilazerte and @jeroen! I've incorporated your edits and comments. @yabellini, I've updated the DeepL Spanish translation.

**Comment by jeroen** [comment]
Thank you! 🚀

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+en los repositorios de R.  Se trata de un **plataforma de publicación de software** que proporciona un
+totalmente automatizada para probar, crear y publicar paquetes de R. También es
+a **meta-repositorio** que ofrece una infraestructura común tanto a particulares como a
+organizaciones para gestionar repositorios de paquetes R personalizados utilizando su propio enfoque.
+la gestión de la publicación y el control de calidad.
+
+R-Universe ya da servicio a decenas de miles de paquetes publicados por más de mil organizaciones.
+organizaciones, y estamos entusiasmados con lo que nos espera. Este año, nuestro objetivo es mejorar las funciones que sirven a grupos que
+están creando repositorios colaborativos a gran escala utilizando la plataforma de R-Universe
+y la API de R-Universe, como la nueva [R-Multiverse](https://r-multiverse.org/). Esto incluye
```
Comment (yabellini) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md:48]:
```suggestion
como el nuevo [R-Multiverse](https://r-multiverse.org/). Esto incluye
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+apoyo para cosas como el desarrollo y la producción ramas de universos, personalizado
```
Comment (yabellini) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md:49]:
```suggestion
apoyo para cosas como ramas de desarrollo y producción de universos,  comprobaciones personalizadas de los paquetes
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+y pasos de aprobación manual para que los propietarios de universos puedan curar y gobernar los universos.
```
Comment (yabellini) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md:50]:
```suggestion
y pasos de aprobación manual para que los propietarios de universos puedan curar y gobernar las
```

File: content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+colecciones de paquetes como mejor les parezca.
+También tenemos previsto ampliar nuestra base de colaboradores mediante más documentación y
+documentación y la colaboración con socios, y mejorar el rendimiento y la fiabilidad a medida que
+creciendo. Estamos deseando trabajar con la comunidad R para hacer de R-Universe una plataforma de aprendizaje en línea.
```
Comment (yabellini) [inline content/blog/2024-12-03-r-universe-r-consortium-tlp/index.es.md:54]:
```suggestion
seguimos creciendo. Queremos trabajar con la comunidad R para hacer de R-Universe una plataforma de primera clase para el software R.
```

**Review by yabellini — APPROVED** [review]
Fixing spanish version.

**Comment by noamross** [comment]
Thanks @yabellini!

I will merge after the R Consortium posts.

**Pull request merged**